### DISCUSSION TITLE
2011/06/17/%E5%85%B3%E4%BA%8E%E7%9B%B8%E5%AF%B9%E8%AE%BA%E6%B2%A1%E6%90%9E%E6%98%8E%E7%99%BD%E7%9A%84%E9%97%AE%E9%A2%98

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#333] [Announcements]
# 2011/06/17/%E5%85%B3%E4%BA%8E%E7%9B%B8%E5%AF%B9%E8%AE%BA%E6%B2%A1%E6%90%9E%E6%98%8E%E7%99%BD%E7%9A%84%E9%97%AE%E9%A2%98

最近研究了一下相对论，准备作为公司每周技术讲座的题目。狭义相对论还是比较容简单的，我基本可以理解。但广义相对论中还有些问题没想清楚，毕竟不是专业研究物理的。

https://blog.qizhen.xyz/2011/06/17/%E5%85%B3%E4%BA%8E%E7%9B%B8%E5%AF%B9%E8%AE%BA%E6%B2%A1%E6%90%9E%E6%98%8E%E7%99%BD%E7%9A%84%E9%97%AE%E9%A2%98

**Comment by ruanqizhen** [comment]
之前博客下的留言：

musella
2011年06月22日 20:25 编辑
牛顿如何看待万有引力 http://wp.me/p2nZ9-5lo

回复
